mapper-runtime/src/main/java/com/datastax/oss/driver/internal/mapper/DaoBase.java

```diff
@@ -143,6 +147,50 @@ public static BoundStatementBuilder populateBoundStatementWithAttributes(
     return builder;
   }
 
+  public BoundStatementBuilder populateBoundStatementWithAttributes(
+      BoundStatementBuilder builder,
+      String profileName,
+      String consistencyLevel,
+      String serialConsistencyLevel,
+      boolean idempotent,
+      int pageSize,
+      long timeout,
+      String keyspace) {
+
+    Duration duration = null;
+    if (timeout != Long.MIN_VALUE) {
+      duration = Duration.ofMillis(timeout);
+    }
```

Comment:
don't we need a similar check and set default for pageSize and idempotent?
Or those two defaults:
```

  int pageSize() default Integer.MIN_VALUE;

  boolean idempotence() default false;
``` 
are safe to pass to the `builder` without conversion to builder neutral values? (like for example for `setExecutionProfileName` the `null` is set if annotation didn't set it explicitly)

Reply:
This has been reworked.

But to answer your original question, if not set these are the default values the builder would use anyway.